<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RecodBook</string>
  </property>
  <widget class="QListWidget" name="listRecordBook">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSave">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>550</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushUpdate">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>550</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushDelete">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>56</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFamily">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>56</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Family:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelSurname">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>56</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Surname:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSurname">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePhone">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPhone">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>56</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>341</width>
     <height>331</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Photo</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushOpenFile">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>450</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
